lib/features/search/presentation/model/simple_search_filter.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/jmap/core/filter/filter.dart';
+import 'package:jmap_dart_client/jmap/core/filter/filter_operator.dart';
+import 'package:jmap_dart_client/jmap/core/filter/operator/logic_filter_operator.dart';
+import 'package:jmap_dart_client/jmap/core/utc_date.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_filter_condition.dart';
+import 'package:model/model.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/model/search/email_receive_time_type.dart';
+import 'package:tmail_ui_user/features/thread/domain/model/search_query.dart';
+
+class SimpleSearchFilter with EquatableMixin {
+  final Set<String> from;
+  final SearchQuery? text;
+  final PresentationMailbox? mailbox;
+  final EmailReceiveTimeType emailReceiveTimeType;
+  final bool hasAttachment;
+  final UTCDate? before;
+
+  SimpleSearchFilter({
+    Set<String>? from,
+    EmailReceiveTimeType? emailReceiveTimeType,
+    bool? hasAttachment,
+    this.text,
+    this.before,
+    this.mailbox,
```

Comment:
how about add the filter with `star` and `read/unread`?

Reply:
Oh, We do not support this feature?

Reply:
designer should add these soon